packages/plugin-ext/src/main/browser/status-bar-message-registry-main.ts

```diff
@@ -13,13 +13,11 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
-import {interfaces} from 'inversify';
+import { interfaces } from 'inversify';
```

Comment:
could we avoid all code style changes ?

Reply:
i think it is auto-format on save which apply project formatting. I guess file was misformatted before.

Reply:
please add to dev meeting to discuss tomorrow

Reply:
yes, was waiting that someone to create entry :-) but I will create it if not created tomorrow